### ISSUE TITLE
Does the program change system settings?

### ISSUE THREAD

**Issue by AnalogKnight** [WyvernTKC/cpuminer-gr-avx2#107]
When the program runs for the first time, the system needs to be restarted to apply MSR. Why? Are there any settings that need to be changed? If the device is no longer mining, how can I clean it up?

Sorry to post this question in the wrong place. I plan to run it on several of my workstations worth tens of thousands of dollars. I just want to know if the program will change the system, and if one day I stop mining, how can I restore it.

**Comment by michal-zurkowski** [comment]
Yes. Restart is required to enable Large Pages. Miner changes policies of the user regarding them and restart of the PC is required to make that functionality to kick in after enabling it.
If you want to manually disable it later (not rly necessary as some apps can benefit from it) you can follow those steps and remove it instead fo adding it to your user, https://docs.microsoft.com/en-us/sql/database-engine/configure-windows/enable-the-lock-pages-in-memory-option-windows

**Comment by AnalogKnight** [comment]
I get it (it seems I got MSR and Large Page wrong), thank you very much for your reply.
Best wishes: D

> 是的。需要重新启动才能启用大页面。Miner 更改用户有关他们的策略，并且需要重新启动 PC 才能使该功能在启用后启动。如果您想稍后手动禁用它（由于某些应用可能从中受益，因此没有必要），您可以按照这些步骤将其删除，而不是将其添加到用户中[，https://docs.microsoft.com/en-us/sql/database-engine/configure-windows/enable-the-lock-pages-in-memory-option-windows](https://docs.microsoft.com/en-us/sql/database-engine/configure-windows/enable-the-lock-pages-in-memory-option-windows)
